src/modules/app/StateType.js

```diff
@@ -55,14 +44,34 @@ export const defaultCitiesState: CitiesStateType = {
   models: []
 }
 
+export type CityContentStateType = {|
```

Comment:
I think we do not need the cityContent object. We can just move all of these into the root. Keeping it flatter makes debugging easier.

Reply:
Or maybe it makes sense because we can lower the scope of the reducer.

Reply:
That's why I introduced it